docs/_archive/bugs/bug-035-error-banner-not-clearable.md

```
@@ -0,0 +1,75 @@
+# BUG-035: Error Banner Not Clearable on Pricing Page
+
+**Status:** Resolved
+**Priority:** P3 - Low
+**Date:** 2026-02-02
+**Resolved:** 2026-02-02
+
+---
+
+## Summary
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)